### DISCUSSION TITLE
How to make height of table fluid

### DISCUSSION THREAD

**Discussion by veronikaEve** [KevinVandy/mantine-react-table#292] [Q&A]
When i try to set the table height to 100vh it stops it's scrollability

```mantineTableContainerProps: {
      ref: tableContainerRef,
      style: { maxHeight: 100vh },
      onScroll: (event) => ....,
    },
```

if i set the paper props to height 100vh and set the table props height to 100% that also stops scrolling. It seems to only allow scrolling when the height is set on the table container as pixels?

**Comment by doelgonzo** [comment]
Running into this now as well.
